bin/citrea/src/guests.rs

```diff
@@ -42,16 +42,16 @@ lazy_static! {
     };
     /// The following 2 are used as latest guest builds for tests that use Bitcoin DA.
     pub(crate) static ref BATCH_PROOF_LATEST_BITCOIN_GUESTS: HashMap<SpecId, (Digest, Vec<u8>)> = {
-        let mut m = HashMap::new();
-
-        m.insert(
-            SpecId::Fork1,
-            (
-                Digest::new(citrea_risc0_batch_proof::BATCH_PROOF_BITCOIN_ID),
-                citrea_risc0_batch_proof::BATCH_PROOF_BITCOIN_ELF.to_vec(),
-            )
-        );
-        m
+        HashMap::from(
+            [
+                #[cfg(feature = "testing")]
+                (SpecId::Genesis, guest!("../../../resources/guests/risc0/devnet/batch-0.elf")),
+                (SpecId::Fork1,
+                    (Digest::new(citrea_risc0_batch_proof::BATCH_PROOF_BITCOIN_ID),
+                    citrea_risc0_batch_proof::BATCH_PROOF_BITCOIN_ELF.to_vec())
+                )
+            ]
+        )
```

Comment:
if we can find another way to test this without depending on testing feature, it would be great. since most tests will have a hard time working around this

Reply:
removed it forcing tests to have both elfs. but added use latest elf config for batch prover which we set to true on all tests except for elf switching test.